frontend/src/components/auth/Login.tsx

```diff
@@ -47,45 +69,85 @@ const Login = (): React.ReactElement => {
   }
 
   return (
-    <div style={{ textAlign: "center" }}>
-      <h1>Login</h1>
-      <form>
-        <div>
-          <input
-            type="email"
-            value={email}
-            onChange={(event) => setEmail(event.target.value)}
-            placeholder="username@domain.com"
-          />
-        </div>
-        <div>
-          <input
-            type="password"
-            value={password}
-            onChange={(event) => setPassword(event.target.value)}
-            placeholder="password"
-          />
-        </div>
-        <div>
-          <button
-            className="btn btn-primary"
-            type="button"
-            onClick={onLogInClick}
-          >
-            Log In
-          </button>
-        </div>
-      </form>
-      <div>
-        <button
-          className="btn btn-primary"
-          type="button"
-          onClick={onSignUpClick}
-        >
-          Sign Up
-        </button>
-      </div>
-    </div>
+    <Flex>
+      <Center flex="1">
+        <VStack>
+          <Image height="13vh" marginBottom="0.5rem" src={logo} />
+          <Text variant="display-sm-sb">Sign in to access courses</Text>
+          <form>
```

Comment:
The form tag is causing weird default behaviour - when I press enter while in the input it attempts to submit the form. This normally would be good, but we don't want it to do that. Either preventing the default behaviour, or perhaps just getting rid of the form tag would be a good idea.